mkdocs.yml

```diff
@@ -329,6 +329,7 @@ plugins:
         "user-guide/administration/installation-extras/docker.md": "user-guide/administration/guides/docker.md"
         "user-guide/administration/installation-extras/health-checks.md": "user-guide/administration/guides/health-checks.md"
         "user-guide/administration/installation-extras/selinux-troubleshooting.md": "user-guide/administration/guides/selinux-troubleshooting.md"
+  - "autorefs"
```

Comment:
I think this is harmless to add but just a note that I just merged #6898 into `develop` which updates `docs/requirements.txt` to similarly fix RTD builds.

Reply:
ack okay ill just purge it from this branch